packages/babel-parser/test/fixtures/typescript/class/index-signature/input.ts

```diff
@@ -3,4 +3,5 @@ class C {
     readonly [x: string]: any;
     static [x: string]: any;
     static readonly [x: string]: any;
+    readonly static [x: string]: any;
```

Comment:
nit: Since this is a test case throwing parsing error, unlike the other tests which are parsed successfully. Can you create a new test `input.ts`? It can be placed in a different folder, tentatively `index-signature-readonly-before-static` but you are free to rename it.